MAUI/ImageEditor/imagefilter.md

```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Image Effect in .NET MAUI Image Editor (SfImageEditor)
 
-Using the image editor control, apply various effects such as Brightness, Hue, Saturation, Contrast, Blur, Opacity, and Sharpen to your image. These effects can be applied using the toolbar or by utilizing the `ImageEffect` method. The `ImageEffect` method consists of two arguments: `ImageEffect` and `EffectValue`. The ImageEffect is an enumeration that includes the following effects:
+Using the image editor control, apply various effects such as Brightness, Hue, Saturation, Contrast, Blur, Opacity, and Sharpen to your image. These effects can be applied using the toolbar or by utilizing the [ImageEffect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEffect.html) method. The [ImageEffect](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ImageEditor.ImageEffect.html) method consists of two arguments: `ImageEffect` and `EffectValue`. The `ImageEffect` is an enumeration that includes the following effects:

```

Comment:
`ImageEffect` => add enum link in 2 places

Reply:
added